dpdk/src/dev.rs

```diff
@@ -239,9 +245,14 @@ impl DevConfig {
                 ..Default::default()
             },
             rxmode: rte_eth_rxmode {
-                // TODO: let user request rx offloads instead of just enabling all supported
-                // offloads.
-                // offloads: self.info.inner.rx_offload_capa,
+                mtu: 1515,
+                mq_mode: RTE_ETH_MQ_RX_RSS,
+                max_lro_pkt_size: 8192,
```

Comment:
Would it make sense to make these parameters more visible (and easier to adjust) by defining consts at the top of the file, rather than “burry” them in the middle of the source code?

Reply:
That is a fair point.  I think we should just expose them as command line params tho.

Reply:
Command-line param sounds good too.

If you don't intend to address it in this PR, can you please at least add a TODO comment so we can find this again and follow up?

Reply:
> If you don't intend to address it in this PR, can you please at least add a TODO comment so we can find this again and follow up?

Please address this.